sql/core/src/test/scala/org/apache/spark/sql/execution/SQLViewSuite.scala

```diff
@@ -910,4 +911,27 @@ abstract class SQLViewSuite extends QueryTest with SQLTestUtils {
       }
     }
   }
+
+  test("SPARK-35685: Prompt recreating view message for schema mismatch") {
```

Comment:
oh I missed it . It's better to move this test to PersistedViewTestSuite

You can fix it in https://github.com/apache/spark/pull/32832